Dockerfiles/5.2.0/secondary_node/Dockerfile

```diff
@@ -1,14 +1,13 @@
-FROM cloudera/clusterdock:centos6.6_nodebase
+FROM clusterdock/topology_nodebase:centos6.6
 
-ADD mapr_core.repo /etc/yum.repos.d/
+ADD mapr_core.repo /etc/yum.repos.d/ 
 ADD mapr_ecosystem.repo /etc/yum.repos.d/
 
+RUN yum -y update
```

Comment:
Why is this needed here and not in the primary_node Dockerfile?

Reply:
Removed.